file_handler.py

```diff
@@ -0,0 +1,131 @@
+import json
+import csv
+import funcs
+
+
+def player_exist(name):
+    """Check if a save exists for player(name)
+
+    Args:
+        name (str): The name of the player
+
+    Returns:
+        bool: True if player exists, False if player does not.
+    """
+    try:
+        with open(f"saves/{name}.json", 'r'):
+            pass
+    except FileNotFoundError:
+        return False
+    else:
+        return True
+
+
+def init_player(name):
+    """Create a player with user given name
+    Builds the skeleton for the json file so the other funcs
+    Will work properly
+
+    Args:
+        name (str): The name of the player
+    """
+    try:
+        with open(f"saves/{name}.json", "w") as f:
+            base_json = {
+                        "boards": {
+                        },
+                        "stats": {
+                            "wins": {
+                                },
+                            "losses": {
+                                }
+                            }
+                        }
+            json.dump(base_json, f)
+    except Exception as e:
+        print(f"Error: {e}")
+        raise
+
+
+def serialize_board(coords, name, board_name):
```

Comment:
https://stackoverflow.com/questions/3316762/what-is-deserialize-and-serialize-in-json

det ser ut att göras mer i denna än bara serialize.